appfl/algorithm/misc.py

```diff
@@ -0,0 +1,83 @@
+import copy
+
+"""Functions for server."""
```

Comment:
I think these functions can be defined under proper classes....

Reply:
misc.py is removed